src/site/_includes/partials/item.njk

```diff
@@ -1,13 +1,19 @@
-{% for slug in section.pathItems %}
-  {% set url = ['', lang, slug, ''] | join('/') %}
+{% for pathItem in section.pathItems %}
+  {% set url = ['', lang, pathItem, ''] | join('/') %}
   {% set item = url | findByUrl %}
-
-  {% if item %}
-
+  {% if pathItem.url %}
+    <li class="w-path-listitem">
+      <a
+        class="w-path-link w-path-link--external"
+        href="{{ pathItem.url }}"
+      >
+        {{ pathItem.title }}
+      </a>
+    </li>
+  {% elif item %}
     {% if item.data.draft %}
```

Comment:
@robdodson in a previous comment you suggested I should refactor this into an if/else format and use `item` instead of `slug`. Since the code already used `item` I opted for `pathItem` instead.

Reply:
Previous comment: https://github.com/GoogleChrome/web.dev/pull/4019#discussion_r507847374